mythril/analysis/potential_issues.py

```diff
@@ -89,7 +89,6 @@ def check_potential_issues(state: GlobalState) -> None:
         except UnsatError:
             continue
 
-        annotation.potential_issues.remove(potential_issue)
```

Comment:
Thanks for the catch! But these potential issues are going to persist in the next transactions, and this will cause a slow down to due increased solving time, so can you deep copy those potential_issues in the for loop?

Reply:
NVM I think just `copy` should do

Reply:
Thank you. I add a new commit, as it uses an ```unsat```  list to record them and finally assign it to the annotation.
Thank your quick response and help very much!